compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -932,70 +932,83 @@ main =
-                  , "  newRights <- grantUserRights \"u1\" [ParticipantAdmin]"
+                  , "  newRights <- grantUserRights u1 [ParticipantAdmin]"
                   , "  newRights === [ParticipantAdmin]"
-                  , "  newRights <- grantUserRights \"u1\" [ParticipantAdmin]"
+                  , "  newRights <- grantUserRights u1 [ParticipantAdmin]"
                   , "  newRights === []"
-                  , "  rights <- listUserRights \"u1\""
+                  , "  rights <- listUserRights u1"
                   , "  rights === [ParticipantAdmin]"
-                  , "  newRights <- grantUserRights \"u1\" [CanActAs p1, CanReadAs p2]"
+                  , "  newRights <- grantUserRights u1 [CanActAs p1, CanReadAs p2]"
                   , "  newRights === [CanActAs p1, CanReadAs p2]"
-                  , "  rights <- listUserRights \"u1\""
+                  , "  rights <- listUserRights u1"
                   , "  rights === [ParticipantAdmin, CanActAs p1, CanReadAs p2]"
-                  , "  revoked <- revokeUserRights \"u1\" [ParticipantAdmin]"
+                  , "  revoked <- revokeUserRights u1 [ParticipantAdmin]"
                   , "  revoked === [ParticipantAdmin]"
-                  , "  revoked <- revokeUserRights \"u1\" [ParticipantAdmin]"
+                  , "  revoked <- revokeUserRights u1 [ParticipantAdmin]"
                   , "  revoked === []"
-                  , "  rights <- listUserRights \"u1\""
+                  , "  rights <- listUserRights u1"
                   , "  rights === [CanActAs p1, CanReadAs p2]"
-                  , "  revoked <- revokeUserRights \"u1\" [CanActAs p1, CanReadAs p2]"
+                  , "  revoked <- revokeUserRights u1 [CanActAs p1, CanReadAs p2]"
                   , "  revoked === [CanActAs p1, CanReadAs p2]"
-                  , "  rights <- listUserRights \"u1\""
+                  , "  rights <- listUserRights u1"
                   , "  rights === []"
-                  , "testUserAlreadyExists = do"
-                  , "  u1 <- createUser (User \"u1\" None) []"
-                  , "  u2 <- createUser (User \"u1\" None) []"
-                  , "  pure ()"
-                  , "testUserNotFound = do"
-                  , "  deleteUser \"nonexistent\""
+                  , "  nonexistent <- validateUserId \"nonexistent\""
+                  , "  expectUserNotFound (listUserRights nonexistent)"
+                  , "  expectUserNotFound (revokeUserRights nonexistent [])"
+                  , "  expectUserNotFound (grantUserRights nonexistent [])"
                   , "  pure ()"
                   ]
                 expectScriptSuccess rs (vr "testUserManagement") $ \r ->
                     matchRegex r "Active contracts: \n"
                 expectScriptSuccess rs (vr "testUserRightManagement") $ \r ->
                     matchRegex r "Active contracts: \n"
-                expectScriptFailure rs (vr "testUserAlreadyExists") $ \r ->
```

Comment:
I think we can also kill it from the proto again with this change. Let’s do that in a separate PR.

Reply:
Ok. Will do in a separate PR.